apps/insights/app/globals.css

```
@@ -24,37 +24,40 @@
-  --sidebar-ring: oklch(0.623 0.214 259.815);
+  --sidebar: #ffffff;
+  --sidebar-foreground: #1a1a1a;
+  --sidebar-primary: #1a1a1a;
+  --sidebar-primary-foreground: #ffffff;
+  --sidebar-accent: #f4f4ef;
+  --sidebar-accent-foreground: #1a1a1a;
+  --sidebar-border: #e8e6de;
+  --sidebar-ring: oklch(70.5% 0.213 47.604);
+  font-family:
+    Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
+    "Segoe UI", sans-serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)